code/src/LeafletProvider/Constants/DrawingTools/Positions.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace LeafletProvider.Constants.DrawingTools {
+    /**
+     * Enum that defines the available Provider Events
```

Comment:
Enum that defines the available positions for the DrawingTools

Reply:
done